spec/index.bs

```diff
@@ -3381,12 +3410,20 @@ A {{GPUProgrammableStageDescriptor}} describes the entry point in the user-provi
         1. If |entry|.{{GPUBindGroupLayoutEntry/type}} is:
             <dl class=switch>
                   : {{GPUBindingType/"sampler"}}
-                  :: the |binding| is a non-comparison sampler
+                  :: the |binding| is a non-comparison sampler, and all the textures it samples
```

Comment:
nit: ideally things in this switch should be more specific, like compareEnable is false instead of "non-comparison", but probably not worth fixing this right now

Reply:
Hmm, upon further consideration, not sure if we'd be able to use that language here.
This spec text talks about the bindings reflected from the shader. So it's sort of an abstract thing, it generally has less information than the bindings on the API side. We could base this off the implicit bind group layout logic, but I think that would be a very wrong thing to do :)

Reply:
Oh, for some reason I thought the "is a non-comparison sampler" was talking about the sampler itself and not the type of the binding variable in the shader.  If the latter, we should say "sampler (not sampler_comparison)" here and "sampler_comparison or sampler" there, or something.